HW28/HW28.cpp

```diff
@@ -0,0 +1,41 @@
+﻿#include <iostream>
+#include <vector>
+#include <thread>
+#include <numeric>
+#include <chrono> 
+//100000000
+
+void sum_vector(const std::vector<int>& vec, int start, int end, long long& result) {
+    result = std::accumulate(vec.begin() + start, vec.begin() + end, 0LL);
+}
+
+int main() {
+    std::vector<int> vec(10000, 1);
```

Comment:
Vector is not set? There's a chance compiler would optimize this so your timing may be wrong

Reply:
No results comparison?

Reply:
Besides, vector size is way too small to have any valid results